service-catalog/components/pulp.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: service-catalog-pulp
+  title: Pulp Cluster
+  description: Pulp is a platform for managing repositories of software packages and making them available to a large number of consumers
+  annotations:
+    operate-first.cloud/logo-url: https://docs.pulpproject.org/pulpcore/en/pulpproject.org_images/pulp_logo_big.png
+    backstage.io/kubernetes-label-selector: "control-plane=controller-manager"
```

Same as above, you want to match all the resources created by the operator here:

```suggestion
    backstage.io/kubernetes-label-selector: "app.kubernetes.io/part-of=pulp"
```